conference.sql

```diff
@@ -30,11 +30,37 @@ CREATE TABLE Sponsor(
     lastName varchar(30),
     email varchar(50),
     emailsSent int,
-    PRIMARY KEY (id)
+    PRIMARY KEY (id),
+    FOREIGN KEY (companyName) REFERENCES SponsorCompany(companyName)
+);
+
+CREATE TABLE SponsorCompany(
+    companyName varchar(30) NOT NULL,
+    numEmailLimit int,
+    ranking varchar(20),
+    PRIMARY KEY (name)
+);
+
+CREATE TABLE JobPostings(
+    jobTitle varchar(30) NOT NULL,
+    jobAddress varchar(50) NOT NULL,
```

Comment:
this should be composite; let's break this down into province and city as per our ER